src/describescreen.cpp

```diff
@@ -428,6 +428,26 @@ void TextWindow::DescribeSelection() {
             double d = (p1.Minus(p0)).Dot(n0);
             Printf(true,  "      distance = %Fi%s", SS.MmToString(d).c_str());
         }
+    } else if(gs.n == 3 && gs.faces == 3) {
+        Printf(false, "%FtTHREE PLANE FACES");
+
+        Vector n0 = SK.GetEntity(gs.face[0])->FaceGetNormalNum();
+        Printf(true,  " planeA normal = " PT_AS_NUM, CO(n0));
+        Vector p0 = SK.GetEntity(gs.face[0])->FaceGetPointNum();
+        Printf(false, "   planeA thru = " PT_AS_STR, COSTR(SK.GetEntity(gs.face[0]), p0));
+
+        Vector n1 = SK.GetEntity(gs.face[1])->FaceGetNormalNum();
+        Printf(true,  " planeB normal = " PT_AS_NUM, CO(n1));
+        Vector p1 = SK.GetEntity(gs.face[1])->FaceGetPointNum();
+        Printf(false, "   planeB thru = " PT_AS_STR, COSTR(SK.GetEntity(gs.face[1]), p1));
+
+        Vector n2 = SK.GetEntity(gs.face[2])->FaceGetNormalNum();
+        Printf(true,  " planeC normal = " PT_AS_NUM, CO(n2));
+        Vector p2 = SK.GetEntity(gs.face[2])->FaceGetPointNum();
+        Printf(false, "   planeB thru = " PT_AS_STR, COSTR(SK.GetEntity(gs.face[2]), p2));
```

Comment:
A [C++11 style range-based for loop.](https://en.cppreference.com/w/cpp/language/range-for)